code/game/gamemodes/cult/talisman.dm

```diff
@@ -168,17 +168,16 @@
 			target.visible_message("<span class='warning'>[target]'s holy weapon absorbs the talisman's light!</span>", \
 								   "<span class='userdanger'>Your holy weapon absorbs the blinding light!</span>")
 		else
-			target.Knockdown(200)
+			target.Stun(200)
 			target.flash_act(1,1)
 			if(issilicon(target))
 				var/mob/living/silicon/S = target
 				S.emp_act(EMP_HEAVY)
 			else if(iscarbon(target))
 				var/mob/living/carbon/C = target
-				C.silent += 5
-				C.stuttering += 15
-				C.cultslurring += 15
-				C.Jitter(15)
+				C.silent += 8
```

Comment:
this is 6 extra seconds(with some variance since life ticks, wooooo)

Reply:
That's the intention, though I recall ticks were 1.7, so closer to 5. Its just clear that while 5 ticks is enough for me to crit people, <10% of cultists follow that style (most don't realize the tome is even that lethal), and for the other 90%+ its not sufficient.